org.mwc.debrief.core/src/org/mwc/debrief/core/ContextOperations/GeneratePasteRepClipboard.java

```diff
@@ -118,57 +120,36 @@ public IStatus undo(IProgressMonitor monitor, IAdaptable info)
 
   private boolean isContentImportable(final String content) {

```

Comment:
it's probably worth introducing a unit test, to verify this passes true/false for the right types of string.

Reply:
done